_projects/nested-languages.md

```diff
@@ -24,16 +24,62 @@ tags:
   - Core
   - Bears
 ---
-Multiple programming languages can coexist in a single source file. coala would
+Generally, a single source file contains code for exactly one programming language.
+But multiple programming languages can also coexist in a single source file. coala would
 benefit from the ability to determine the programming language of each part of
 a file. If we have e.g. a PHP file, it is likely that we'd have chunks of html
 inside a few of them. We would like coala to be able to still run meaningful
 analysis on those files.
 Other examples include:
 
-* awk inside bash
-* JSON inside Javascript
-* HTML, LaTeX or others inside Jinja2
+* JSON inside Javascript is valid in some cases, for example JSONFormatBear for coala.
+* Web templating - languages such as PHP or JSP files mix code into HTML.
+* Cython, it is a superset of the Python language that additionally supports
+  calling C functions and declaring C types on variables and class attributes.
+* We can call Python code from Java using Jython, which is a Python implementation in Java.
+* reStructuredText and Markdown language specify different language for codeblock.
+
+  For example
+
+````
+```python
+s = "Python syntax highlighting"
+print s
+```
+````
+OUTPUT
```

Comment:
check your indentation of the rendered version

Reply:
But whats the problem in that?
 I used 4 backslash to show the markdown syntax